modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -21,69 +21,94 @@ ifeval::["{context}" == "installing-restricted-networks-vmc"]
 endif::[]
 
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
-= Sample install-config.yaml file for an installer-provisioned VMware vSphere cluster
+= Sample `install-config.yaml` file for an installer-provisioned VMware vSphere cluster
 
-You can customize the install-config.yaml file to specify more details about
-your {product-title} cluster's platform or modify the values of the required
-parameters.
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
 
-[source,yaml]
+You must specify at least one failure domain for your {product-title} cluster, so that {product-title} can evenly spread virtual machine nodes across domains or groups on a vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and so on. 
+
+The following example shows a VMware vSphere cluster configuration file with multiple data centers defined for a VMware vCenter. 
+
+[source,yaml,subs=quotes]
```

Comment:
```suggestion
[source,yaml]
```
This is causing the XML error. normally, the build script converts this:
```
    folder: "/<datacenter_name>/vm/<folder_name>/<subfolder_name>" <12>
```
to this:
```
    folder: "/&lt;datacenter_name&gt;/vm/&lt;folder_name&gt;/&lt;subfolder_name&gt;" <co xml:id="CO935-14"/>
```
with `subs=quotes` enabled, it takes the `<` and `>` symbols literally and interprets them as xml tags.

Reply:
Hi @bscott-rh :pray: . Thank you so much, this recommended change worked a treat. You have solved the biggest technical headache I've had in some time. :bowing_man: